packages/env/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### New Feature
 
 - The `.wp-env.json` coniguration file now accepts `port` and `testsPort` options which can be used to set the ports on which the docker instance is mounted.
+- You may now override the directory in which wp-env creates generated files with the `WP_ENV_SOURCE` environment variable. (The default directory is `~/.wp-env/`.)
```

Comment:
```suggestion
- You may now override the directory in which `wp-env` creates generated files with the `WP_ENV_SOURCE` environment variable. (The default directory is `~/.wp-env/`.)
```

Shouldn't this also be on top since it came after the other feature?

Reply:
sure :)